docs/Earn/liquid-staking/dot/dot-liquid-staking-mechanics.md

```diff
@@ -0,0 +1,183 @@
+   Upon claim confirmation, the user interacts with Polkadot.js extension to send a request to the ANKR backend to get a notarization signature. To send the request, the user needs to include a signature in it. The signature is created by signing the marshalled message `<Bytes>StakeFi Signed Message:\n[POLKADOT_PUBLIC_KEY_HEX]\n[ETHEREUM_ADDRESS]\n[CURRENT_TIMESTAMP]</Bytes>`. The resulting signature is included in the request.
+
+6. The `claim` object in the reply contains the following fields: 
+
+  ```{
+  "claim": {
+    "address": "string",
+    "status": "ACTIVE"|"CLAIMED"|"EXPIRED",
+    "data": {
+      "network": 0,
+      "claimId": "0xdeadbeef...",
+      "claimBeforeBlock": 123321123,
+      "amount": "12.1234",
+      "nativeAmount": "12123400000000",
+      "signature": "0xdeadbeef...",
+      "ethAddress": "0xdeadbeef..."
+    },
+    "createdTimestamp": 0,
+    "expiresTimestamp": 0,
+    "tokenAddress": "0xdeadbeef...",
+    "loanId": 0 # 0 for staking, >0 for crowdloans
+  },
+  "tokenAddress": "0xdeadbeef..."
+  }
+  ```
+6. To check there are no errors, before the actual claim, the user runs a call to `PolkadotPool`:
+  ```
+PolkadotPool::isClaimValid(
+  address token, 
+  uint256 claimId, 
+  uint256 claimBeforeBlock, 
+  uint256 amount, 
+  address to, 
+  bytes memory signature
+) external view returns (bool)
+  ```
+
+  If the call returns true, the user sends a transaction to claim aDOTb:
+
+  ```
+PolkadotPool::claimBonds(
+  address bond, 
+  uint256 claimId, 
+  uint256 amount, 
+  uint256 claimBeforeBlock, 
+  address account, 
+  bytes memory signature
+) external
+  ```
+
+  All fields are taken from the `claim` payload from Step 5.
+
+7. Once the transaction is successful, `PolkadotPool` mints aDOTb to the user's wallet. The balance shows on the staking dashboard and in Metamask (only if user added the token) via a call to `aDOTb::balanceOf(address)`.
+
+### Daily exchange rate updates
+
+To update the ratio, the ANKR backend:
+
+1. Collects information about all stakes in progress and rewards to be received by the end of the current validation period.
+
+2. Collects all aDOTb claims.
+
+3. Collects information about the total supply of aDOTb from `aDOTb`.
+
+4. Recalculates the aDOTb share — the aDOTb exchange rate. The share is an analog of aETHc (which historically was aETH), but for Polkadot. For example, user has 1 share, but due to ratio change, aDOTb balance grows daily.
+
+## Unstaking Flow
+
+1. User calls `PolkadotPool::burnBond(address bond, uint256 amount, bytes calldata polkadotRecipient)` where `bond` is the aDOTb contract address, `amount` is the amount to burn, and `polkadotRecipient` is the user’s address on Polkadot.
+
+2. ANKR backend catches a `TokensBurned` event, emitted by `PolkadotPool` and saves the unstaking request to the database.
```

Comment:
caches instead of catches

Reply:
AFAIK, it doesn't cache, it listens to events and react to the ones that it is programmed to react to.

Reply:
where does the "caches" come from?